charts/newrelic-logging/templates/daemonset.yaml

```diff
@@ -118,8 +139,15 @@ spec:
           volumeMounts:
             - name: fluent-bit-config
               mountPath: /fluent-bit/etc
-            - name: var
-              mountPath: /var
+            - name: logs
+              mountPath: {{ .Values.fluentBit.linuxMountPath | default "/var" }}
+              {{- if and (.Values.fluentBit.persistence) (ne .Values.fluentBit.persistence.mode "hostPath") }}
+              readOnly: true
+              {{- end }}
```

Comment:
Ah, now I understand why you kept "/var", is it because we also need to keep /var/lib for symlinks? I would suggest to just mount the needed directories, [like the Fluent Bit Helm chart does](https://github.com/fluent/helm-charts/blob/main/charts/fluent-bit/values.yaml#L444-L449), then.

Reply:
Mentioned already in another comment but that way we'll need to add a "skipVarLib" flag or something for GKE autopilot, I think it's more flexible to do it that way. Could be people using other path than var for logs (that's why we allow `fluentBit.path` in the configuration I believe)